tests/introspection/ApiTypoTest.cs

```diff
@@ -1105,6 +1105,10 @@ public void ConstantsCheck ()
 #endif
 				default:
 					if (fi.Name.EndsWith ("Library", StringComparison.Ordinal)) {
+						// NFC is currently not available on iPad
+						if (fi.Name == "CoreNFCLibrary" && UIDevice.CurrentDevice.UserInterfaceIdiom == UIUserInterfaceIdiom.Pad)
```

Comment:
That file is shared with XM so `UIDevice` won't build for Mac tests. You either use the product subclass, e.g. `iOSApiTypoTest.cs`, or use an `#if` around the non-portable code

Also for performance that test is only run on simulators and when the latest OS version is selected. That normally means (makefiles & xharness) that the iPad is not used - not that it makes it incorrect to fix :)

Reply:
🤣 now that the bots have reported back, `UIDevice` is also a bit more limited in the watch profile